app/views/search/single_facility.html.erb

```diff
@@ -1,62 +1,122 @@
 <% provide(:title, "Single Facility") %>
 <div id="content" class="interior">
   <a id="content" href="#content" class="sr-only">Content starts here.</a>
-  <%= render 'layouts/breadcrumbs' %>
-</div> 
```

Comment:
I had moved the breadcrumb into a partial. Can we use this partial instead (so put this back in and remove lines 5-14 I believe)?  (In your defense, I made this change after you originally wrote the file, so if you just copied it over, it would have deleted this without you knowing).